core/src/main/scala/org/genivi/sota/core/transfer/DeviceUpdates.scala

```diff
@@ -98,24 +98,28 @@ object DeviceUpdates {
     }
   }
 
-  /**
-    * Find the [[UpdateRequest]]-s for:
-    * <ul>
-    *   <li>the given device</li>
-    *   <li>and for which [[UpdateSpec]]-s exist with Pending or InFlight [[UpdateStatus]]</li>
-    * </ul>
-    */
```

Comment:
Fair point. If they are becoming useless, we should probably look at removing them all or updating them. Having them being sporadically present doesn't seem very satisfying either.